ci-operator/step-registry/ipi/install/ipi-install-chain.yaml

```diff
@@ -3,5 +3,6 @@ chain:
   steps:
   - ref: ipi-install-rbac
   - ref: ipi-install-install
+  - ref: ipi-install-post-install
```

Comment:
I don't think we want a step that's called `post-install`. The step should have a clear task (setup support secret) and be called accordingly.

Reply:
Yea I wanted to make this step to be able to hold different kind of actions that are needed before the tests run. Your suggestion makes sense but we will have to inject this step in all jobs that need it.

Reply:
Yeah but that's kind of the point of the steps - looks like not all jobs need this, so the ones that do can just include this step.